addons/api/package.json

```diff
@@ -41,6 +41,7 @@
     "@ember/string": "^3.0.1",
     "@ember/test-helpers": "^2.9.3",
     "@embroider/test-setup": "^2.1.1",
+    "@faker-js/faker": "^8.0.2",
```

Comment:
What is the reason for adding faker here, it was not here before, right?

Reply:
Oh I see before it was on line 67 and now we move it to 44?

Reply:
yeah I am not exactly sure why it moved. It just did.

Reply:
It's because it's supposed to be alphabetical ordering so it re-ordered it

Reply:
Oh yeah. I would have expected to see that more often though so I didn't think to check for the order.